tool/site/client-react/src/components/radioField.tsx

```diff
@@ -0,0 +1,26 @@
+import React, {FC} from "react"
+import {Radios} from "nhsuk-react-components"
+import {Field} from "formik"
+
+export interface RadioFieldProps {
+  name: string
+  label: string
+  fieldRadios: Array<Radio>
+  error?: string
+}
+
+interface Radio {
+  value: string
+  text: string
+  defaultChecked?: boolean
```

Comment:
Given that there can only be one default checked radio button, I wonder whether it makes more sense to represent this as a `defaultValue` prop in `RadioFieldProps`. Then set `defaultChecked` on the Radio if the value equals the default value

Reply:
makes sense, will update